src/complex.rs

```diff
@@ -59,6 +59,14 @@ impl<T: Clone + Num> Complex<T> {
     }
 }
 
+impl<T: Clone + Num> Complex<T> {
+    /// Returns `i`
+    #[inline]
+    pub fn i() -> Complex<T> {
+        Self::new(T::zero(), T::one())
+    }
+}
```

Comment:
Why did you make this a separate `impl` block from the one above?
I'd also be a little more descriptive, like "Returns the imaginary unit."